test/files/neg/t3115b.check

```diff
@@ -0,0 +1,9 @@
+t3115b.scala:17: warning: class C in package p is deprecated
```

Comment:
that's not entirely accurate...

Reply:
I didn't think yet to update the messaging.

Reply:
Edit: "Oh so now the error messages have to be accurate."